mne/viz/tests/test_topomap.py

```diff
@@ -68,7 +68,7 @@ def test_plot_topomap():
 
     ev_bad = evoked.pick_types(meg=False, eeg=True, copy=True)
     ev_bad.pick_channels(ev_bad.ch_names[:2])
-    ev_bad.plot_topomap(times=ev_bad.times[:2] - 1e-6)  # auto, should plot EEG
+    ev_bad.plot_topomap(times=ev_bad.times[1:3] - 1e-6)  # auto, plots EEG
```

Comment:
you should loosen the check. Values should be between

tmin - 1. / (2. * sfreq) and tmax + 1 /( 2. * sfreq)